presentation/virtualKeyboard.brs

```diff
@@ -3,41 +3,92 @@ REM @title                   BrightSign Virtual Keyboard
 REM @author                  Lee Dydo
 REM @company                 BrightSign LLC.
 REM @date-created            06/06/2016
-REM @date-last-modified      07/05/2016
+REM @date-last-modified      06/07/2016
```

Comment:
Why may this date be older than the prior version? Also if anything, should it be updated to "today"?

Reply:
ah yeah that's an artifact from before. will update